src/octoprint/plugins/pluginmanager/static/js/pluginmanager.js

```diff
@@ -1108,10 +1216,75 @@ $(function () {
                     {url: _.escape(url), name: _.escape(name)}
                 );
             }
+
+            if (self.multiInstallRunning()) {
+                workTitle =
+                    _.sprintf("[%(index)d/%(total)d] ", {
+                        index:
+                            this.multiInstallInitialSize() -
+                            self.multiInstallQueue().length,
+                        total: this.multiInstallInitialSize()
+                    }) + workTitle;
+            }
+
             self._markWorking(workTitle, workText);
 
             var onSuccess = function (response) {
                     self.installUrl("");
+                    if (response.hasOwnProperty("queued_installs")) {
+                        self.queuedInstalls(response.queued_installs);
+                        if (typeof self.installQueuPopup !== "undefined") {
```

Comment:
That should rather be `self.installQueuePopup`, shouldn't it? It's consistently wrong, but I'd rather get rid of that typo now 😅

Reply:
oops, adjusted